src/control/cmd/daos/container.go

```diff
@@ -351,6 +353,73 @@ func (cmd *containerCreateCmd) Execute(_ []string) (err error) {
 	return nil
 }
 
+type containerSerializeCmd struct {
+	daosCmd
+
+	Source string `long:"src" short:"s" description:"container to be serialized" required:"1"`
+	Output string `long:"output-path" short:"o" description:"path to output serialized HDF5 files" required:"0"`
+}
+
+func (cmd *containerSerializeCmd) Execute(_ []string) error {
+	ap, deallocCmdArgs, err := allocCmdArgs(cmd.log)
+	if err != nil {
+		return err
+	}
+	defer deallocCmdArgs()
+
+	ap.src = C.CString(cmd.Source)
+	defer freeString(ap.src)
+	if cmd.Output != "" {
+		ap.output_path = C.CString(cmd.Output)
+		defer freeString(ap.output_path)
+	}
+
+	ap.c_op = C.CONT_SERIALIZE
+	rc := C.cont_serialize_hdlr(ap)
+	if err := daosError(rc); err != nil {
+		return errors.Wrapf(err,
+			"failed to serialize %s -> %s", cmd.Source)
```

Comment:
@daltonbohning yeah I believe so but I just removed printing the output dir since it isn't always specified. So it will just print "failed to serialize <Sourrce>"